anomalib/models/dfm/README.md

```diff
@@ -1,20 +1,34 @@
+<!-- markdownlint-disable line-length -->
```

Comment:
Can we not configure the line length in the settings? I feel line length of 80 is too restricting.

Reply:
That's my feeling as well

Reply:
There is a possibility to change the line-length such as [here](https://github.com/markdownlint/markdownlint/blob/master/example/new_style_example.rb). However, we need to create our own `.mdlrc` file in the repo. As far as I understand, tox or pyproject.toml files are yet to be supported.

Reply:
I am in favour of adding a mldrc file just to keep the formatting consistent but it it is a lot of efforts then maybe we can create a separate PR for that.

Reply:
I agree. We could use the mldrc file for now, until it supports the `pyproject.toml` file. This would be a lot cleaner than disabling and enabling the check in every file.

Reply:
ok, I've now added the `.mldrc` file and `markdownlint.rb` files for the configuration.